src/storage/write_ahead_log/log_serializer_task.cpp

```diff
@@ -59,17 +59,19 @@ void LogSerializerTask::LogSerializerTaskLoop() {
   } while (run_task_);
   // To be extra sure we processed everything
   Process();
-  NOISEPAGE_ASSERT(flush_queue_.empty(), "Termination of LogSerializerTask should hand off all buffers to consumers");
+  NOISEPAGE_ASSERT(disk_flush_queue_.empty(),
```

Comment:
I think we probably want to add an assert for the `replication_flush_queue_` as well.